src/scripts/settings/webSettings.js

```diff
@@ -2,7 +2,17 @@ let data;
 
 function getConfig() {
     if (data) return Promise.resolve(data);
-    return fetch('/config.json?nocache=' + new Date().getUTCMilliseconds()).then(function (response) {
+    return fetch('/config.json?nocache=' + new Date().getUTCMilliseconds()).then(response => {
```

Comment:
Watch out, as this still won't handle servers with a baseUrl set.

Not blocking though, it can be done later.

Reply:
That was intentional because I was running into some issues with that and wanted to look into it more. I haven't forgotten it though, I think I'll look into the plugin URLs at the same time.